src/Compilers/Test/Resources/Core/MetadataTests/Invalid/Signatures/munge.csx

```diff
@@ -0,0 +1,14 @@
+string file = Args[0];
+string hexSearch = Args[1];
+string hexReplace = Args[2];
+
+File.WriteAllBytes(
+    Args[0],
+    BitConverter.ToString(File.ReadAllBytes(Args[0]))
```

Comment:
Nit against my own repro code here. I meant to use the file local instead of repeating Args[0] twice.